### DISCUSSION TITLE
Managing filters for email notification

### DISCUSSION THREAD

**Discussion by dec-bethe** [HewlettPackard/POSH-HPOneView#550] [Q&A]
I try to create a filter that will only notify me for events I'm interested in. To roll out that filter expression and recipient list on approx. 60 OV appliance I like to use the CMDlet  "Add-OVSmtpAlertEmailFilter". I found following issues:
1. With that CMDlet I can only "add" new filters but not modify them, e.g. add an additional filter expression, add new recipient, etc.
2. I found no CMDlet for modifying or deleting an alert email filter
If I had distributed an alert email filter, how ca I manage it on so many appliance? What about the global dashboard, that would be nice if it were possible from it.

**Comment by ChrisLynchHPE** [comment]
Only the Add Cmdlet exists right now.  The library will need to be enhanced to add that support.  Or a custom script would need to be developed to perform the actions you want to in the meantime.

As for Global Dashboard, this module only supports OneView, not Global Dashboard.  We have another project that provides a PowerShell library to manage Global Dashboard.  But it is behind in development (we actually had an open source project maintained by an HPE partner transfer it to us.)

**Comment by ChrisLynchHPE** [comment]
I added the following Cmdlets

* Get-OVSmtpEmailFilter/Get-HPOVSmtpEmailFilter
* Add-OVSmtpEmailFilter/Add-HPOVSmtpEmailFilter
* Set-OVSmtpEmailFilter/Set-HPOVSmtpEmailFilter
* Remove-OVSmtpEmailFilter/Remove-HPOVSmtpEmailFilter

To the following releases:

* [5.50.2680.1687](https://www.powershellgallery.com/packages/HPEOneView.550/5.50.2680.1687) 
* [5.40.2681.1725](https://www.powershellgallery.com/packages/HPEOneView.540/5.40.2681.1725)
* [5.30.2682.3811](https://www.powershellgallery.com/packages/HPEOneView.530/5.30.2682.3811)
* [5.20.2684.2161](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2684.2161)
* [5.00.2685.1865](https://www.powershellgallery.com/packages/HPOneView.500/5.0.2685.1865)
